cli.js

```diff
@@ -33,5 +28,11 @@ if (flags.columns.length > 0) {
   options.columns = flags.align.map(align => ({ align }))
 }
 
-// write results to stdout
-console.log(convert(input, options))
+stdin.then(function( stdin ){
```

Comment:
These are just style nitpicks if you can get to them. Could you make this an arrow function? I also don't necessarily like shadowing variables if we can help it so let's rename the param to `input` since that's freed up now.

```js
stdin.then(input => {
```

Reply:
I don't mind style nitpicking. I'd do the same.